kebechet/managers/version/version.py

```diff
@@ -203,6 +203,20 @@ def run(  # type: ignore
 
         trigger: BaseTrigger
 
+        if (
+                self.parsed_payload
+                and utils._is_merge_event(self.parsed_payload)
+                and utils._is_version_pr(self.parsed_payload)
+                and utils._is_release_pr(self.parsed_payload)
+        ):
+            version =  utils._get_version(self.parsed_payload)
+            tag = f"v{version}"
+            with cloned_repo(self) as repo:
+                _LOGGER.info(f"Creating Tag of version {tag}")
+                _LOGGER.info(repo.git.execute(["git","tag", f"v{version}", f"{utils._get_merge_commit_sha(self.parsed_payload)}"]))
```

Comment:
Interesting that  ogr doesn't have a function in 
 built to create tag https://packit.github.io/ogr/services/base.html#ogr.services.base.BaseGitService

Reply:
Creating a tag is more of a function of `git`, OGR does have an option to create a release from a tag though. Do we need to create a release to kick of the proper CI/CD workflows? If we do it should be called after creating the tag.